cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -1414,6 +1444,60 @@ define(['jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers', 'common/j
                     );
                 });
 
+                describe('Show correctness setting set as expected.', function() {
+                    var setShowCorrectness = function(show_correctness) {
+                        $('input[name=show-correctness][value=' + show_correctness + ']').click();
+                    }, expectWarningShown = function(shown) {
+                        expect($('.show-correctness .tip-warning').is(':visible')).toBe(shown);
+                    };
+
+                    describe('Show correctness set by subsection metadata.', function() {
+                        $.each(['always', 'never', 'past_due'], function(index, show_correctness) {
```

Comment:
@pomegranited It would look a bit better if it were using [Array.forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) instead of jQuery. But  if it doesn't work out of the box (because of old browsers running tests) - that's ok to keep it like this.